provider/datagen/Cargo.toml

```diff
@@ -197,6 +213,6 @@ skip_feature_sets = [["use_icu4c"], ["use_wasm"]]
 skip_optional_dependencies = true
 # Always the experimental deps because we want to all-or-nothing them
 # and the nothing case is already tested in regular check CI
-always_include_features = ["experimental_components"]
+always_include_features = ["experimental_components", "blob_input"]
```

Comment:
why put `blob_input` here?

Reply:
to reduce complexity, and because the binary gives compiler warnings when both features are disabled

Reply:
What warnings?

Reply:
*compiler errors

Reply:
`compile_error!("Feature `bin` requires either `blob_input` or `provider`")`

Reply:
ok... please leave a comment explaining this, and we may want to add this to cargo-all-features, something similar to `skip_feature_sets` but with the opposite behavior